### DISCUSSION TITLE
Quais seriam as rotas?

### DISCUSSION THREAD

**Discussion by eletroswing** [diego3g/rsxp-2023#15] [General]
# As rotas
Um passo importante a ser definido seriam as rotas para 'mokar' em uma api como json-server ou o [mswjs](https://mswjs.io/), como citado na [discussão #2](https://github.com/diego3g/rsxp-2023/discussions/2).

## Exemplo do que me refiro
*/ingresso*
   > PATCH : vincula o ingresso enviado a sua conta
   > GET: pega o ingresso vinculado da conta autenticada
   > DELETE: remove os ingressos veiculados a conta

**Comment by LuizHAP** [comment]
Acho que primeiro de tudo, é criar as rotas usando terminologia em inglês. O que acha?

**Reply by eletroswing** [reply]
então, so um exemplo esse do /ingressos, mas acredito que o correto seria algo como /passcard ou /card, ou um endpoint /feed, por exemplo

**Comment by henriqgoncalvs** [comment]
Acho que não é possível usar o MSW, pq ele é baseado em service workers que só funcionam num ambiente de browser. Porém dá pra pensar em usar ele se a gente for escrever testes (unitários ou de integração)

Mas pra mockar o ideal seria json-server mesmo

**Comment by LuizHAP** [comment]
Podemos usar nos testes de integração sim. Mas seria legal criar os
contratos primeiro para dps desenhar

**Reply by henriqgoncalvs** [reply]
Exato, tem uma discussão focada em testes. Acho que esse assunto vai evoluir mais por lá, mas concordo contigo

**Comment by diego3g** [comment]
Galera, podemos criar juntos isso na próxima live! Acho que aqui pelo Github fica ruim falar sobre esse tipo de coisa tão técnica sem termos nem um start do projeto! O que acham?

**Reply by LucasMedeiros7** [reply]
Acredito que fazendo uma live para esclarecer alguns pontos vai ficar melhor mesmo. Tem alguma previsão de próxima live para botar esses pontos em ordem?

**Comment by diego3g** [comment]
Galera criei um board simples de tarefas por enquanto via issues pra irmos nos organizando: #49 

Vou fechar essa discussão por enquanto pra termos mais espaço pra novas discussões 🤗
